scripts/insert-results-for-framework.py

```diff
@@ -0,0 +1,30 @@
+"""
```

Comment:
Probably `scripts/insert-results-for-framework.py` -> `scripts/insert-framework-results.py` so as to keep everything the same.